packages/react-storage/src/components/StorageBrowser/controls/hooks/__tests__/useTitle.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { useControlsContext } from '../../../controls/context';
+import { useTitle } from '../useTitle';
+
+jest.mock('../../../controls/context');
+
+describe('useTitle', () => {
+  const data = {
+    title: 'ShinyNewTitle',
+  };
+
+  const mockUseControlsContext = jest.mocked(useControlsContext);
+
+  afterEach(() => {
+    mockUseControlsContext.mockReset();
+  });
+
+  it('returns Title data', () => {
+    mockUseControlsContext.mockReturnValue({ data });
+
+    expect(useTitle()).toStrictEqual({
+      title: 'ShinyNewTitle',
+    });
+  });
+
+  it('returns empty string for children if title is missing from data', () => {
```

Comment:
The test says it returns empty string but actually returns undefined. The hook is also not returning children 😛